api/overdrive.py

```diff
@@ -746,50 +749,23 @@ def json(self):
         return json.loads(self.content)
 
 
-class DummyOverdriveAPI(OverdriveAPI):
+class DummyOverdriveAPI(MockOverdriveAPI, OverdriveAPI):
```

Comment:
It is confusing to have two mock API classes. Are you planning to combine them eventually?